modules/jira/index.js

```diff
@@ -126,8 +130,9 @@ module.exports = class Jira extends BaseModule {
     return CALLBACK_ID;
   }
 
-  createRoutes(app) {
-    this.jiraCreate.createRoutes(app, CALLBACK_ID);
+  async createRoutes(app) {
```

Comment:
how often is this function called?

we might want to fetch the project list every time the command is called so that the list is always up to date? or at least every time the slack client is started

Reply:
Should only be called once

Reply:
You could cache them for a day or so. I use a small lib for that:

https://github.com/konecnyna/slack-cat/blob/cbe6354e9bc0c62d8fb580d5cb022096f65a3f51/modules/plus/index.js#L86